### PR TITLE
Scss starter file (main.scss) and sass compiler

### PR DESCRIPTION
**Pull request by El-amiir**

### CODE REVIEW

File: public/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "public",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "scss": "node-sass --watch assets/sass/main.scss -o assets/css"
```
Comment (mohamed-abdul-fattah) [inline public/package.json:8]: This is a development script. We need a script the compiles the scss files without watch.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /.idea/
 /configs.php
 /vendor/
-/.phpunit.result.cache
\ No newline at end of file
+/.phpunit.result.cache
+/public/node_modules
```
Comment (mohamed-abdul-fattah) [inline .gitignore:5]: Nitpick.
Add a line feed to the end of the file. You can configure your editor to automatically add a line feed on save.

File: public/assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
+/* Cairo Google font */
```
Comment (mohamed-abdul-fattah) [inline public/assets/css/main.css:1]: This an output file. Please, gitignore this file as it could be generated.

File: public/package.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (mohamed-abdul-fattah) [inline public/package.json:11]: Out license is MIT.

File: public/package.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "public",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "scss": "node-sass --watch assets/sass/main.scss -o assets/css"
+  },
+  "author": "",
```
Comment (mohamed-abdul-fattah) [inline public/package.json:10]: You're the author. Add your email in the following format `username <username@email.com>`

File: public/package.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "public",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (mohamed-abdul-fattah) [inline public/package.json:7]: Please, remove this script as there is no tests yet.

**Review by mohamed-abdul-fattah — COMMENTED** [review]
Kindly, check my comments in a later PR. Please, move the package.json and the node modules to the project root directory.

File: public/assets/css/main.css (hunk 3 above)
Comment (El-amiir) [reply public/assets/css/main.css:1]: it could be generated , but if gitignore it you cant keep track new changes

Comment (mohamed-abdul-fattah) [reply public/assets/css/main.css:1]: No need for the new changes as this is an output file, and the real changes that we need to track is the Sass file.

**Pull request merged**